docs/setup/keadm.md

```diff
@@ -96,70 +96,308 @@ KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
-  ...
+    ```bash
+    cd metrics-server
+    ```
 
-  ...
-  modules:
-    edgeStream:
-      enable: true
-  ...
-  ```
+    Make the docker image:
 
-6. Restart all the cloudcore and edgecore.
+    ```bash
+    make container
+    ```
 
-### Support Metrics-server in Cloud
-1. The realization of this function point reuses cloudstream and edgestream modules. So you also need to perform all steps of *Enable `kubectl logs` Feature*.
+    Check if you have this docker image:
 
-2. Since the kubelet ports of edge nodes and cloud nodes are not the same, the current release version of metrics-server(0.3.x) does not support automatic port identification, so you need to manually compile the image yourself now.
+    ```bash
+    docker images
+    ```
+
+    |                  REPOSITORY                           |                    TAG                   |   IMAGE ID   |     CREATE     |  SIZE  |
+    |-------------------------------------------------------|------------------------------------------|--------------|----------------|--------|
+    | gcr.io/k8s-staging-metrics-serer/ metrics-serer-amd64 | 6d92704c5a68cd29a7a81bce68e6c2230c7a6912 | a24f71249d69 | 19 seconds ago | 57.2MB |
+    | metrics-server-kubeedge                               |                 latest                   | aef0fa7a834c | 28 seconds ago | 57.2MB |
 
-```bash
-git clone https://github.com/kubernetes-sigs/metrics-server.git
-cd metrics-server
-make container
-docker images
-docker tag 2d9eb6e0d887 metrics-server-kubeedge:latest
-```
+
+    Make sure you change the tag of image by using its IMAGE ID to be compactable with image name in yaml file.
+
+    ```bash
+    docker tag a24f71249d69 metrics-server-kubeedge:latest
+    ```
 
 3. Apply the deployment yaml. For specific deployment documents, you can refer to https://github.com/kubernetes-sigs/metrics-server/tree/master/manifests.
 
+    Before you deploy metrics-server, you have to make sure that you deploy it on the node which has apiserver deployed on. In my case, that is my master node. As a consequence, I need to make master node schedulable by the following command:
+
+    ``` shell
+    kubectl taint nodes --all node-role.kubernetes.io/master-
+    ```
+   
+    Then, in the deployment.yaml file, I must specify that metrics-server is deployed on master node.
+    (I used hostname as the marked label.)
+    In `metrics-server-deployment.yaml`
+    ``` yaml
+        spec:
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  #Specify which label in [kubectl get nodes --show-labels] you want to match
+                  - key: kubernetes.io/hostname
+                    operator: In
+                    values:
+                    #Specify the value in key
+                    - charlie-latest
+    ```
+
+    The deployment YAML files can be found in this repository:
```

Comment:
There is no need for a specific yaml file, just the original way.

Reply:
Oh, I will remove it.
(Since I am a newbie and have no idea how to create it in an original way, I thought it is easier to have an example.)